packages/data/tests/unit/helpers/metric-format-test.js

```diff
@@ -17,6 +17,9 @@ test('It formats metrics appropriately', function(assert) {
   assert.equal(metricFormat({metric: {name: 'foo', longName: 'Foo'}, parameters: {p: 1}}, 'Bar'),
     'Bar (1)',
     'Formats metric with parameters');
+  assert.equal(metricFormat({ metric: { name: 'foo', longName: 'Foo' }, parameters: { p: { defaultValue:1 } } }, 'Bar'),
```

Comment:
Are we sure about this object shape?
```js
{ metric: { name: 'foo', longName: 'Foo' }, parameters: { p: { defaultValue:1 } } }
```

Also, can we verify the fix with an acceptance test?

Reply:
So this shape does look correct.  However I still need to figure out how to produce this for an acceptance test.